github_rules/github_repo_created.py

```diff
@@ -1,5 +1,5 @@
 def rule(event):
-    return event.get("action") == "repo.created"
+    return event.get("action") == "repo.create"
```

Comment:
Confirmed: https://docs.github.com/en/enterprise-server@3.4/admin/monitoring-activity-in-your-enterprise/reviewing-audit-logs-for-your-enterprise/audit-log-events-for-your-enterprise#repo-category-actions